src/Our.Umbraco.StackedContent/Web/UI/App_Plugins/StackedContent/js/stackedcontent.controllers.js

```diff
@@ -36,6 +54,36 @@
             $scope.model.value.splice(idx, 1);
         }
 
+        $scope.copyToLocalStorage = function (evt, idx) {
+            var stackedContentItem = Object.assign({}, $scope.model.value[idx]);
+            stackedContentItem.key = "";
+            stackedContentItem.$$hashKey = "";
+
+            if (validateModel(stackedContentItem)) {
+                window.localStorage.setItem("StackedContentCopy", JSON.stringify(stackedContentItem));
+                notificationsService.success("Stacked Content", "Copied to clipboard.");
+                return;
+            } else {
+                notificationsService.error("Stacked Content", "Sorry, something went wrong.");
+            }
+        }
+
+        $scope.pasteFromLocalStorage = function (evt, idx) {
+            var stackedContentItem = JSON.parse(window.localStorage.getItem("StackedContentCopy"));
+            if (!stackedContentItem) {
+                notificationsService.error("Stacked Content", "You need to copy content first.");
+                return;
+            }
+            if (validateModel(stackedContentItem)) {
+                $scope.overlayConfig.event = evt;
+                $scope.overlayConfig.data = { model: stackedContentItem, idx: idx, action: "add" };
+                $scope.overlayConfig.show = true;
+                return;
```

Comment:
Is this auto opening the edit dialog? If so, I'd probably not bother doing that.